inspirehep/utils/export.py

```diff
@@ -122,10 +122,34 @@ def _get_slac_citation(self):
                     cite_element = field['value'].upper()
                     cite_line = '%%CITATION = ' + cite_element + ';%%'
             if not cite_element and self.record['report_numbers']:
-                cite_element = self.record['report_numbers'][0]['value'].upper()
+                cite_element = self.record[
+                    'report_numbers'][0]['value'].upper()
                 cite_line = '%%CITATION = ' + cite_element + ';%%'
         else:
             cite_element = str(self.record['recid'])
             cite_line = '%%CITATION = ' + 'INSPIRE-' + \
                 cite_element + ';%%'
         return cite_line
+
+    def _get_citation_number(self):
+        """Returns how many times record was cited. If 0, returns nothing"""
+        from invenio_ext.es import es
+        from inspirehep.utils.citations import Citation
+        import time
+        today = time.strftime("%d %b %Y")
+        record = Citation(es.get(index='hep',
```

Comment:
Here what you need is to fetch only the record you are currently formatting and accessing the field `citation_count `

No need to use `from inspirehep.utils.citations import Citation` as that will recompute the citations.